### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by KeisukeShima**
## What kinds of PR

- [x] Improve feature

## What is this PR
Create dependabot for GitHub Actions

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
```
Comment (kenji-miyake) [inline .github/dependabot.yml:6]: Do we not need to set `open-pull-requests-limit`?

Comment (KeisukeShima) [reply .github/dependabot.yml:6]: Looking at the documentation, I think we can keep the default settings. (Up to 5 pull requests)

> By default, Dependabot opens a maximum of five pull requests for version updates. Once there are five open pull requests, new requests are blocked until you merge or close some of the open requests, after which new pull requests can be opened on subsequent updates. Use open-pull-requests-limit to change this limit. 

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#open-pull-requests-limit

Comment (kenji-miyake) [inline .github/dependabot.yml:6]: Probably we don't need all the comments because these are the template.
Which page did you refer to?

Comment (KeisukeShima) [reply .github/dependabot.yml:6]: OK. I will remove comments.
I created this file from Insights tab --> Dependency graph --> Dependabot tab --> Create config file

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**